### DISCUSSION TITLE
Windows - JavaScript

### DISCUSSION THREAD

**Discussion by kikolak** [xseman/bysquare#7] [Q&A] [answered]
Hello, 

Is it possible to run this library on windows? I would like to use it with JavaScript on the front-end. Thank you.

**Comment by xseman** [comment]
Can you elaborate on what you mean by "run this library on windows"?
As of now, only nodejs is supported.

The ultimate goal is to be able to run the library on any client, ideally with
WebAssembly, so there will be room for other runtimes like browsers, deno, etc.

**Reply by xseman** [reply, marked as answer]
The library now fully supports JavaScript in any runtime, so compatibility is guaranteed for any platform.
